cnv/cnv_users_guide/cnv-cloning-vm-disk-into-new-datavolume-block.adoc

```diff
@@ -0,0 +1,29 @@
+[id="cnv-cloning-vm-disk-into-new-datavolume-block"]
+= Cloning a virtual machine disk into a new block storage DataVolume
+include::modules/cnv-document-attributes.adoc[]
+:context: cnv-cloning-vm-disk-into-new-datavolume-block
+toc::[]
+
+You can clone the PersistentVolumeClaim (PVC) of a virtual machine disk into
+a new block DataVolume by referencing the source PVC in your DataVolume configuration
+file.
+
+.Prerequisites 
+
+* You may need to xref:../../cnv/cnv_users_guide/cnv-preparing-cdi-scratch-space.adoc#cnv-defining-storageclass-in-cdi-configuration_cnv-preparing-cdi-scratch-space[define a StorageClass or prepare CDI scratch space]
```

Comment:
Can this be a step instead of a prereq? If not, please flip it around to say something like "If you require scratch space according to the matrix, create it."